docs/src/IconButton.doc.js

```diff
@@ -1,419 +1,862 @@
+    icon="visit"
+    role="link"
+    target="blank"
+    href="https://www.pinterest.com"
+  />
+</Tooltip>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Size"
+      description={`IconButton is available in 5 fixed sizes:
 
-card(
-  <Example
-    id="ref"
-    name="ref"
-    defaultCode={`
+1. \`xl\` (56px)
+    Extra large IconButtons should be used sparingly and only in places where the UI has a case for an extra-large IconButton.
+2. \`lg\` (48px)
+    Large is the only size that should be used on Pinner surfaces.
+3. \`md\` (40px)
+    Medium is the size used on more dense UI such as business surfaces or internal tools.
+4. \`sm\` (32px)
+    Small Icon button should be used sparingly and only in places where the UI is very dense.
+5. \`xs\` (24px)
+    Use sparingly and only in places where the UI is very dense or has a case for an extra-small IconButton as they can be hard to see for people with visual impairments.
+
+Use padding sparingly. Padding options are 1-5 representing the padding in boints units. Combine the \`padding\` with \`size\` options for custom icon/button size ratios. If omitted, padding is derived from the default padding for each \`size\` prop.`}
+    >
+      <CombinationNew size={['xs', 'sm', 'md', 'lg', 'xl']}>
+        {({ size }) => (
+          <IconButton
+            accessibilityLabel={`Example size ${size}`}
+            bgColor="lightGray"
+            icon="add"
+            iconColor="darkGray"
+            size={size}
+          />
+        )}
+      </CombinationNew>
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Color"
+      description={`IconButton can be presented in combinations of icon and background colors. In the absence of combinations, for each \`iconColor\` or \`bgColor\` value, a default paired value is assigned.
+
+Follow these guidelines for \`iconColor\`
+
+1. Red ("red"). High emphasis, used for primary actions.
+2. Dark Gray ("darkGray"). Medium emphasis, used for secondary actions.
+3. Light Gray ("lightGray"). Low emphasis when placed on white backgrounds, used for tertiary actions, and medium emphasis, used for secondary actions when placed on dark backgrounds.
+4. White ("white"). Used in a dark mode scheme or over a dark-colored background creating better visibility.
+
+Follow these guidelines for \`bgColor\`
+
+1. Red ("red"). High emphasis, used for primary actions.
+2. Dark Gray ("darkGray"). Medium emphasis, used for secondary actions.
+3. Transparent Dark Gray ("transparentDarkGray"). Medium emphasis, used for secondary actions, usually above a colored background.
+4. Light Gray ("lightGray"). Low emphasis when placed on white backgrounds, used for tertiary actions, and medium emphasis, used for secondary actions when placed on dark backgrounds.
+5. White ("white"). Used when there is a need of an IconButton over an image or colored background to provide a better contrast and visibility.
```

Comment:
"to provide better contrast and visibility"

Reply:
DONE